src/main/java/com/flightstats/hub/metrics/MetricsConfig.java

```diff
@@ -22,4 +27,38 @@
     int reportingIntervalSeconds;
     String role;
     String team;
+
+    private static String parseAppVersion() {
```

Comment:
I need to think about a better way to do this;... we can possibly inject `HubVersion` into the `MetricsConfigBuilder` and then it could be mocked? Hmm. I might also move this into the builder class if that's the only place it's being used.

Reply:
I'll work on cleaning this up

Reply:
I think I'm using a provider for the s3 verifier config I extracted out, so that'd be neat for consistency's sake.